### PR TITLE
Fixes #37512 - Drop all OVAL related things

### PR DESCRIPTION
**Pull request by adamruzicka**
```
# grep -Ri oval . | grep -v test/data
./COPYING:removal in certain cases when you modify the work.)  You may place
```

Might be a bit heavy handed, opening just as a draft for now.

### CODE REVIEW

File: lib/smart_proxy_openscap/fetch_scap_file.rb [hunk 1]
```diff
@@ -24,22 +24,18 @@ def download_path
...
-        "api/v2/compliance/oval_policies/:policy_id/oval_content"
       end
     end
 
     def file_name(policy_id, digest)
       case @type
       when :scap_content, :tailoring_file
         "#{policy_id}_#{digest}.xml"
-      when :oval_content
-        "#{digest}.oval.xml.bz2"
```
Comment (ofedoren) [inline lib/smart_proxy_openscap/fetch_scap_file.rb:37]: We could also remove this from the test data: https://github.com/theforeman/smart_proxy_openscap/blob/master/test/data/oval-results.xml.bz2

**Review by ofedoren — COMMENTED** [review]
Thanks, @adamruzicka, couldn't find here anything else except for two things inline.

File: settings.d/openscap.yml.example [hunk 2]
```diff
@@ -31,6 +31,3 @@
 # Affects sending reports to Foreman (directly and from spool) and fetching scap content or tailoring file
 # for distribution to clients
 #:timeout: 60
-
-# Directory where OpenSCAP OVAL content bzipped XML are stored
-#:oval_content_dir: /var/lib/openscap/oval_content
```
[the remark this answers is not in the document]
Comment (adamruzicka) [reply settings.d/openscap.yml.example:36]: Last time I checked this wasn't exposed in the installer/puppet modules, but I'll check it again.

[the remark this answers is not in the document]
Comment (adamruzicka) [reply settings.d/openscap.yml.example:36]: The config template was never updated to include this https://github.com/theforeman/puppet-foreman_proxy/blob/master/templates/plugin/openscap.yml.erb

File: lib/smart_proxy_openscap/fetch_scap_file.rb (hunk 1 above)
Comment (adamruzicka) [reply lib/smart_proxy_openscap/fetch_scap_file.rb:37]: Dropped this and one more

**Review by ofedoren — APPROVED** [review]
Thanks, @adamruzicka !

**Pull request merged**